package.json

```
@@ -8,7 +8,8 @@
   ],
   "scripts": {
     "build": "turbo run build --",
-    "dev": "turbo run dev --parallel",
+    "staging": "turbo run staging --parallel",
+    "local": "turbo run local --parallel",
```

no need to change these @packplusplus `dev` is always local. environments are only a thing when running the `build` command.

```suggestion
    "dev": "turbo run dev --parallel",
```